pkg/agent/openflow/pipeline.go

```diff
@@ -2155,48 +2155,48 @@ func (c *client) genPacketInMeter(meterID binding.MeterIDType, rate uint32) bind
 func (c *client) generatePipeline() {
 	bridge := c.bridge
 	c.pipeline = map[binding.TableIDType]binding.Table{
-		ClassifierTable:    bridge.CreateTable(ClassifierTable, spoofGuardTable, binding.TableMissActionDrop),
-		arpResponderTable:  bridge.CreateTable(arpResponderTable, binding.LastTableID, binding.TableMissActionDrop),
-		conntrackTable:     bridge.CreateTable(conntrackTable, conntrackStateTable, binding.TableMissActionNone),
-		EgressRuleTable:    bridge.CreateTable(EgressRuleTable, EgressDefaultTable, binding.TableMissActionNext),
-		EgressDefaultTable: bridge.CreateTable(EgressDefaultTable, EgressMetricTable, binding.TableMissActionNext),
-		EgressMetricTable:  bridge.CreateTable(EgressMetricTable, l3ForwardingTable, binding.TableMissActionNext),
-		l3ForwardingTable:  bridge.CreateTable(l3ForwardingTable, l2ForwardingCalcTable, binding.TableMissActionNext),
-		l3DecTTLTable:      bridge.CreateTable(l3DecTTLTable, l2ForwardingCalcTable, binding.TableMissActionNext),
+		ClassifierTable:    bridge.CreateTable(ClassifierTable, spoofGuardTable, binding.TableMissActionDrop, "ClassifierTable"),
```

Comment:
We already define table names in FlowTables, could we use that?

Reply:
FlowTables actually use table ID as the key string, and it is not the readable name string. Do you think I should also change FlowTable key to use the name string defined here @jianjuns ?

Reply:
I meant you can get table name by FlowTables[ClassifierTable]. Of course I like to hear if you have a better idea to build the relations between table, name, and ID.

Reply:
Get your point. I would prefer to add "name" property in binding.Table, and we could leverage the property directly after OVS is connected to Antrea Agent (no need to back read "name" value from agent/openflow). Then we could provide a function "GetName" in interface Table. Since we have a cached Table object in pkg/agent/openflow, we could read the name property from Table object. For FlowTables defined in pkg/agent/openflow, we could remove it if it is only used to map the tableID and tableName. For convenience, it should be better to change the pipeline map to a cache object, which supports to get binding.Table using TableIDType or tableName as indexer. What do you think?